16-slices/exercises/16-internals-backing-array-fix/main.go

```diff
@@ -46,7 +46,7 @@ func main() {
 
 	// DON'T TOUCH THE FOLLOWING CODE
 	//
-	// This code changes the elements of the nums
+	// This code changes the elements of the mine
```

Comment:
It was meant to show that the code changes the nums slice as well as the mine slice because we're changing the backing array.

Reply:
Ok removing it.